dtfs-central-api/api-test.jest.config.js

```diff
@@ -4,6 +4,6 @@ module.exports = {
   globalTeardown: './test-teardown-globals.js',
   collectCoverageFrom: ['src/**/*.{js,}'],
   coverageDirectory: 'generated_reports/coverage/api-test',
-  testMatch: ['**/*.api-test.js'],
+  testMatch: ['**/*.test.js', '**/*.api-test.js'],
```

Comment:
@AlexBramhill can you please create a unit test config (and a unit test command, which we'll need to add to the GHAs) instead please?

Reply:
Updated